libcxx/test/libcxx/feature_test_macro/get_dialect_versions.sh.py

```diff
@@ -0,0 +1,53 @@
+# RUN: %{python} %s %{libcxx-dir}/utils %{libcxx-dir}/utils/data/feature_test_macros/test_data.json
+# ===----------------------------------------------------------------------===##
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# ===----------------------------------------------------------------------===##
+
+import sys
+
+sys.path.append(sys.argv[1])
+from generate_feature_test_macro_components import feature_test_macros
+
+
+def test(output, expected):
```

Comment:
Let's use Python's `unittest` module instead.

Reply:
I did a short test with that module but that seems to eat the command line arguments of the test. So for now I left the test unchanged.